### ISSUE TITLE
Design Systems: UX Research - 2023 Agenda

### ISSUE THREAD

**Issue by rachelita2** [hackforla/design-systems#479]
Template
### **[1/12/2023] Agenda and Notes**

**- [ROLL CALL](https://docs.google.com/spreadsheets/d/1JtJGxSpVQR3t3wN8P5iHtLZ-QEk-NtewGurNTikw1t0/edit#gid=1199107562) -**
Present: 

**- RECURRING ITEMS -**
- [ ] Review any issues that are in the [new issue approval column](https://github.com/hackforla/design-systems/projects/1#column-15133576)
- [ ] Accountability and support check.
- [ ] Review assignments

**- AGENDA -**
- [ ] New items for meeting discussion

**- MEETING NOTES -**
- Quick notes about meeting for review

**- ACTION ITEMS -**
- [ ] Items to do

**- RESOURCES -**
[Figma file](https://www.figma.com/file/5NacfOm45QlrWgr6wWEg4k/Design-System-Template-for-HfLA?node-id=2%3A54&t=r0qPxeKsx29UQhk6-0)
[HfLA Figma Kit](https://www.figma.com/file/XIy3Dciciej4Q6A1TXGMS1/HfLA-Figma-Kit?node-id=2%3A16&t=SAwrdulQxK1A00A7-0)
[Old Design Systems Research](https://www.figma.com/file/UvUJ0zT2EEIOoDQVdn6ApN/Design-Systems-Team-Figma?node-id=0%3A1&t=iCGOD877P5TUJBSz-0)

**Comment by allthatyazz** [comment]
### **1/18 Agenda and Notes**

**- [ROLL CALL](https://docs.google.com/spreadsheets/d/1JtJGxSpVQR3t3wN8P5iHtLZ-QEk-NtewGurNTikw1t0/edit#gid=1199107562) -**
Present: Rachel, Eva, Faezeh, Gala, Zeynep, Eli

**- AGENDA -**
- [ ] @GalaStojnic  to present a [draft of ](https://docs.google.com/document/d/1u95uWRK_IVvVbChKSdrudCBKXF7oM_mXPfP2H0nGEUI/edit?usp=sharing)questions that researchers can use in the upcoming user testing
- [ ] Team members share their thoughts and questions


**- MEETING NOTES -**
- A few clarifying questions that were answered in the meeting
  - There are different versions of the product. **Which version of the product are we going to test with the users**? **We are testing the released version which is the first official release called HfLA Figma Kit v1.0**  
  - **Who are the users? They are designers at HfLA projects with varying levels of experience** ( the common persona is recent bootcamp graduates who are looking for their first UX design role.)
 - Some of the designers' questions are very specific and detailed and there is no way to address those through contextual inquiries. For example, the following questions are based on some assumptions that have not been validated yet. Such questions can project a negative attitude on users while in reality, the users may interact with the product without any problem.
   -   Q1: Will putting an additional screenshot visual be helpful to users for better understanding or just be too much to look at and overwhelm the user? Problem: The assumption is user has a hard time comprehending the "standards". The hypothesis is that Solution A ( having a screenshot) would make the user's pain better. We have to first validate the assumption. But the question does not target the underlying user's pain, yet it goes straight to the solution. 
   - Q2: Would it be helpful to make “Icon Guidelines” for users? Or give more examples? Problem: It is not possible to test without showing the user two alternatives side by side
   - Q3: Do the “Descriptive Names” make sense or should there be a more extensive example for them? Problem: Too specific 
 - The research itself can be iterative: 
   - Different rounds of study can be designed.
   - During the initial round of contextual inquiry we can determine areas to focus more on in the 2nd round. 
   - A/B testing is not recommended for the first round.   
- Providing the users with some scenarios (tasks) can be a better solution. 
- Researchers need to start their own document outlining the design of the research study. The draft prepared by Gala and the following discussions can be used as a base for the new document.


**- ACTION ITEMS -**
- [ ] @fuzjan81 will start a new document so researchers can collaborate on the same document 
- [ ] Researchers will workshop with designers in the next all-hand  


**- RESOURCES -**
[Figma file](https://www.figma.com/file/5NacfOm45QlrWgr6wWEg4k/Design-System-Template-for-HfLA?node-id=2%3A54&t=r0qPxeKsx29UQhk6-0)
[HfLA Figma Kit](https://www.figma.com/file/XIy3Dciciej4Q6A1TXGMS1/HfLA-Figma-Kit?node-id=2%3A16&t=SAwrdulQxK1A00A7-0)
[Old Design Systems Research](https://www.figma.com/file/UvUJ0zT2EEIOoDQVdn6ApN/Design-Systems-Team-Figma?node-id=0%3A1&t=iCGOD877P5TUJBSz-0)

**Comment by allthatyazz** [comment]
### **1/25 Notes**

**- [ROLL CALL](https://docs.google.com/spreadsheets/d/1JtJGxSpVQR3t3wN8P5iHtLZ-QEk-NtewGurNTikw1t0/edit#gid=1199107562) -**
Present: Rachel, Eva, Faezeh, Gala, Zeynep, Eli

**- MEETING NOTES -**
- Researchers collaborated on the plan. The progress has been documented in the drive.

**Comment by allthatyazz** [comment]
### **2/1 Agenda and Notes**

**- [ROLL CALL](https://docs.google.com/spreadsheets/d/1JtJGxSpVQR3t3wN8P5iHtLZ-QEk-NtewGurNTikw1t0/edit#gid=1199107562) -**
Present: Rachel, Eva, Faezeh, Gala, Zeynep, Eli

**- AGENDA -**
- [ ] Questions on the recruitment 
  - [ ] Do we need a screener? If yes, what are the requirements?
  - [ ] So we want to send out the screener to the UX-UI Slack channel or target selected teams and post on their channel?
  - [ ] What should be the message we post there? 
- [ ] Team members edit the research plan


**- MEETING NOTES -**

**Comment by allthatyazz** [comment]
# Task Items for executing the research 
- Who are the participants? Designers
- Snowball sample? 
- Contact each team's PM  
- Do we need to first figure out the availability of the participants 
- [ ] Yas to compile a list of active projects + Slack channel 
- [ ] Draft an outreach message
- [ ] Screener?! 
- [ ] Create a spreadsheet / or a google doc 
- [ ] Matching the availability of our researchers 
- [ ] Find/research an auto-transcribing tool?

**Comment by allthatyazz** [comment]
# **2/22  Notes**

**- [ROLL CALL](https://docs.google.com/spreadsheets/d/1JtJGxSpVQR3t3wN8P5iHtLZ-QEk-NtewGurNTikw1t0/edit#gid=1199107562) -**
Present:  Yas Rachel, Eva, Faezeh, Gala, Zeynep, Eli

- Rachel is stepping down from the Lead position to focus on another project which lasted for 8 weeks. She will have reduced responsibilities for the time being but still try to provide consultancy for the team. 
- Team progress updates:  Interviews conducted with P1 and P2. P2 sounds very knowledgeable and shared a lot of good tips. Faezeh and Gala will have their interviews by the end of the week. Eva's point of contact from Lucky Parking says they never used Figma before so we decided to approach another team. The PM suggested #civictechjobs instead but Eva can follow up with the #HomeUniteUS team as well. 
- After rewatching Eli's interview and hers, Zeynep discovered some **areas for improvement** and shared them with the team.
   - When conducting an interview, it's important to ensure that the participant shares their entire browser when screen sharing and not just a single tab.
   
  - During the screen sharing, it was noticed that the zoom menu was obstructing some parts of the screen, causing difficulty for the participant to view certain elements. Therefore, it is recommended to ask the participant to move it so that it does not interfere with their view.
  - For Likert scale questions, it's important to show the actual scale to the participants instead of just asking them to rate on a numerical scale. This will allow them to have a better understanding of the questions and rate them more accurately.
    - During the survey, it's important to ensure that participants can see the actual scale so they build **a spatial perception.**  So the scale is about not just having the numerical values. When there is a visual element, participants can see the labels on the scale which provide additional context and meaning to the numerical values, making it easier for participants to interpret and rate the questions accurately. 
    - During the meeting, Eva raised a concern about the two Likert questions related to the first scenario, which were intended to measure helpfulness and difficulty. She pointed out that the current scales were confusing for users because the positive and negative values were working in opposite directions for each question.

After discussing this issue as a team, we made the decision to remove all Likert questions except for the one related to the difficulty of the first task. This question was considered meaningful for the product strategy. Instead of the removed Likert questions, we decided to use open-ended questions. For instance, we could ask participants about the illustration in a more open-ended style.

# Action items to move forward 
- [ ]  Researchers to collaborate on a slide deck to share preliminary insights with the team during the next all-hands meeting on February 27.
**PM Advice:** The team should not be overly concerned about the aesthetics of the slides. Instead, they should focus on providing simple bullet points and relevant screenshots to support their findings.

- [ ] Long-term action item:  Researchers will prepare a report on user insights to share with the design team for product improvement. The report will highlight the key findings, recommendations, and potential solutions for enhancing the user experience based on the research studies conducted during this phase.

- [ ] Long-term action item: Researchers will prepare a separate report focused on the learnings from the research studies. This report will evaluate the steps we take to come up with the research plan, results, and analysis, as well as any challenges encountered and recommendations for future research initiatives. The report will serve as a valuable resource for future researchers and stakeholders interested in building upon the findings of this phase.

**Comment by zeaslan** [comment]
### **3/1 Agenda and Notes**

**- [ROLL CALL](https://docs.google.com/spreadsheets/d/1JtJGxSpVQR3t3wN8P5iHtLZ-QEk-NtewGurNTikw1t0/edit#gid=1199107562) -**
Present: 


**- AGENDA -**
- [ ] Go over the status of the interviews 
- [ ] Discuss the insights from the all-hands meeting 
- [ ] Discuss and determine the method of the in-depth analyses 
- [ ] Go over the time frame of the analyses and the final research report


**- MEETING NOTES -**

**Comment by GalaStojnic** [comment]
### **[3/8] Agenda and Notes**

**- [ROLL CALL](https://docs.google.com/spreadsheets/d/1JtJGxSpVQR3t3wN8P5iHtLZ-QEk-NtewGurNTikw1t0/edit#gid=1199107562) -**
Present: 

**- AGENDA -**
- [ ] Update on the last participant’s interview (take a look at the insights slides if they’re updated with the new participant/ or otherwise discuss the insights)
- [ ] Plan affinity diagramming analysis and discuss the timeline
- [ ] Go over the materials Eli combined in the email for the reference 
- [ ] Other issues?


**- MEETING NOTES -**


**- ACTION ITEMS -**



**- RESOURCES -**
https://www.figma.com/file/rbW89yoCha3ojI7kxaqvil/Interviews---Affinity-Diagram?t=jHyxqqPYkAI35pN4-0
https://www.figma.com/file/rbW89yoCha3ojI7kxaqvil/Interviews---Affinity-Diagram?t=JY1VgudgeldNqY0b-0
https://www.figma.com/file/l911DJDfDo0Yr70lagjSmx/Usability-test-analysis-FigJam

**Comment by GalaStojnic** [comment]
### **[3/15] Agenda and Notes**

**- [ROLL CALL](https://docs.google.com/spreadsheets/d/1JtJGxSpVQR3t3wN8P5iHtLZ-QEk-NtewGurNTikw1t0/edit#gid=1199107562) -**
Present: 

**- AGENDA -**
- [ ] Affinity Diagramming exercise
- [ ] Organize sticky notes into broader categories (~30 min)
- [ ] Determine subcategories (~10 minutes)
- [ ] Summarize the categories (~10 minutes)
- [ ] Determine priorities (~5 minutes)
- [ ] Wrap-up - report draft discussion (~5 minutes)


**- MEETING NOTES -**
- Quick notes about meeting for review


**- ACTION ITEMS -**
- [ ] Items to do


**- RESOURCES -**
https://www.figma.com/file/Tl7hsak9UMPO6N1cNoSbVp/Figma-Kit-Post-Launch-User-Testing---Affinity-Diagram?node-id=0%3A1&t=2J9Zi7n9y02Jl94o-0
[Figma file](https://www.figma.com/file/5NacfOm45QlrWgr6wWEg4k/Design-System-Template-for-HfLA?node-id=2%3A54&t=r0qPxeKsx29UQhk6-0)
[HfLA Figma Kit](https://www.figma.com/file/XIy3Dciciej4Q6A1TXGMS1/HfLA-Figma-Kit?node-id=2%3A16&t=SAwrdulQxK1A00A7-0)

**Comment by GalaStojnic** [comment]
### **[March 22] Agenda and Notes**

**- [ROLL CALL] -**
Present: 

**- AGENDA -**
- [ ] Updating people who missed the previous meeting with the existing affinity diagram and the report draft (~5 minutes);
- [ ] Work on subcategories for the affinity diagram (~30 minutes);
- [ ] Determine key insights; put them down on the diagram  (~15 minutes);
- [ ] Discuss the report writing plan (~5 minutes)


**- MEETING NOTES -**
- Quick notes about meeting for review


**- ACTION ITEMS -**
- [ ] Items to do

**Comment by allthatyazz** [comment]
### **3/29 Agenda and Notes**

**- [ROLL CALL](https://docs.google.com/spreadsheets/d/1JtJGxSpVQR3t3wN8P5iHtLZ-QEk-NtewGurNTikw1t0/edit#gid=1199107562) -**
Present: 


**- AGENDA -**
FYI: Planning and brainstorming for the next 2 months. 
- [ ] What comes next?
- [ ] Option A: Onboarding Survey
- [ ] Option B: Research artifacts?
   - [ ] User Personas
   - [ ] User Journey Map
- [ ] Option C: Start planning for testing the DS starter template  


**- MEETING NOTES -**
- Quick notes about meeting for review


**- ACTION ITEMS -**
- [ ] Items to do

**Comment by GalaStojnic** [comment]
### **[04/12] Agenda and Notes**

**- [ROLL CALL](https://docs.google.com/spreadsheets/d/1JtJGxSpVQR3t3wN8P5iHtLZ-QEk-NtewGurNTikw1t0/edit#gid=1199107562) -**
Present: Gala, Eli, Faezeh, Zeynep, Yas

**- AGENDA -**
- [x]  Update on option A, i.e. the survey project (from Monday meeting), make sure everyone’s on the same page;
- [x] Talk about option C (i.e. early UXR for DS starter) and discuss the suggestion to do focus groups with CoP designers.


**- MEETING NOTES -**
- After ruling out option A as a priority, the team agreed to focus on option C (early research for Design Starter). Discussed wanting to do focus groups (agreed on this), but also agreed that we need more info from the Design Team on what we need to do (i.e. what kind of research they need, to determine the goals of the research to be done). 


**- ACTION ITEMS -**
- [ ] Create a shared folder with resources and materials on focus groups;
- [ ] Direct message design team leads to ask about the product stage and what kind of research they need/ want;
- [ ] See if there are focus groups resources in CoP drive;
- [ ] See if there are any focus groups experts in CoP and whether they'd like to be a guest on one of our research meetings.

**Comment by Leah0996** [comment]
@allthatyazz

**Comment by GalaStojnic** [comment]
### **[05/03] Agenda and Notes**

**- [ROLL CALL](https://docs.google.com/spreadsheets/d/1JtJGxSpVQR3t3wN8P5iHtLZ-QEk-NtewGurNTikw1t0/edit#gid=1199107562) -**
Present: Zeynep, Gala, Eli, Faezeh, Khanh (guest) 


**- RECURRING ITEMS -**
- [ ] Review any issues that are in the [new issue approval column](https://github.com/hackforla/design-systems/projects/1#column-15133576)
- [ ] Accountability and support check.
- [ ] Review assignments


**- AGENDA -**
- [x] Khanh demoing component library (short demo). Discuss what the designers want to understand from early research.
- [x] Discuss timeline with Khanh;
- [x] Discuss with the research method for component library (we talked about focus groups. Do we still agree this is the way to go?)
- [x] Update the team on the Naming research Michelle & Sandra want us to do;
- [x] Eli’s updates on his review of the previous DS research.


**- MEETING NOTES -**
- Quick notes about meeting for review
- Eli updated us with his research progress (review and organizing of previous DS research and outcomes), shared the document with the team and requested their feedback; 
- Khanh presented the demo of component library to the research team. Timeline and potential research were discussed. Ideally, they'd release the prototype by the end of June. Research team could potentially do focus groups (or other kind of research) with the released product, or before it's release as well. One option is to do a more focused series of research tasks that would target specific aspects of the current version of the product (before) the release, and then do an omnibus style research for the released product (e.g. focus groups).
- Zeynep, Gala & Eli met with Michelle for the content team and discussed research needed for the naming activity; we will do a survey with developers; the timeline is 2 weeks


**- ACTION ITEMS -**
- [ ] Decide on research steps for the current product; decide on whether we'd have more specific research techniques targeting specific aspects of the product, or a more general usability test (or focus group) for the product as a whole;
- [ ]  If more specific research questions is what we will go for, list them and decide on techniques to answer them;
- [ ] Create a survey for the naming activity; start a google doc to draft the survey;
- [x] Ask Yas whether HfLA has a preferred platform for surveys (e.g. Qualtrics);
- [x] Go over Eli's research document and provide feedback (https://docs.google.com/document/d/1_D75cxpd2jUm_vJdGB29HstfrrTQm5nRzgB0OAUOsME/edit)

**Comment by GalaStojnic** [comment]
### **[05/10] Agenda and Notes**

**- [ROLL CALL](https://docs.google.com/spreadsheets/d/1JtJGxSpVQR3t3wN8P5iHtLZ-QEk-NtewGurNTikw1t0/edit#gid=1199107562) -**
Present: Zeynep, Eli, Gala, Faezeh, Yas


**- RECURRING ITEMS -**
- [ ] Review any issues that are in the [new issue approval column](https://github.com/hackforla/design-systems/projects/1#column-15133576)
- [x] Accountability and support check.
- [ ] Review assignments


**- AGENDA -**
- [x] Talk the platform for naming survey (Maze or something else?)
- [x] Work on the survey draft created by Zeynep; come up with questions/ introductory blurb **https://docs.google.com/document/d/1batkrOPFJTDL10XiU5X87YaPYAntESicUOooi3DSSNc/edit**


**- MEETING NOTES -**
- Yas will get subscriptions for both Maze and UsabilityHub for the design systems team. Will decide on which one to use once the survey template is ready. 
- Yas explained that the timeline for the research study does not necessarily need to be as tight as initially planned.
- Yas proposed a different structure for the naming survey, where users would be asked about their thoughts on the name alternatives before being provided with the actual product details. The research team agreed that this would be a valuable approach.
- Once the survey is ready, the research team will advertise it on the General slack channel of HfLA 
- Blurb describing the product and the pictures of the product will be created in collaboration with the design and content teams. The research team will work on the initial draft of the blurb. 

- Eli and Yas went over the Research Studies Analysis document and Eli edited the "Outcomes" of the research studies in light of Yas' input. 

**- ACTION ITEMS -**
- [x] Zeynep will edit the research plan and the relevant issue in light of the decisions made during the meeting
- [x] @elimarietta @fuzjan81 and @GalaStojnic will review the research plan materials and the team will communicate over Slack to divide the workload. 


**- RESOURCES -**
(https://docs.google.com/document/d/1batkrOPFJTDL10XiU5X87YaPYAntESicUOooi3DSSNc/edit)
)https://app.maze.co/home
https://docs.google.com/document/d/1hhLR1NFzEn8SS9NbN9PiF3v_ReprUxuPGM-FSValwQc/edit

**Comment by GalaStojnic** [comment]
### **[05/17] Agenda and Notes**

**- [ROLL CALL](https://docs.google.com/spreadsheets/d/1JtJGxSpVQR3t3wN8P5iHtLZ-QEk-NtewGurNTikw1t0/edit#gid=1199107562) -**
Present: 


**- RECURRING ITEMS -**
- [ ] Review any issues that are in the [new issue approval column](https://github.com/hackforla/design-systems/projects/1#column-15133576)
- [ ] Accountability and support check.
- [ ] Review assignments


**- AGENDA -**
- [ ] Brief recap of the all hands meeting issues;
- [ ] Working on the draft for the naming survey https://docs.google.com/document/d/1batkrOPFJTDL10XiU5X87YaPYAntESicUOooi3DSSNc/edit


**- MEETING NOTES -**
- Quick notes about meeting for review


**- ACTION ITEMS -**
- [ ] Items to do
